src/Raven.Server/ServerWide/Commands/ETL/AddEtlCommand.cs

```diff
@@ -64,6 +66,21 @@ public override void UpdateDatabaseRecord(DatabaseRecord record, long etag)
         {
             Add(ref record.RavenEtls, record, etag);
         }
+
+        public override void AssertLicenseLimits(ServerStore serverStore, DatabaseRecord databaseRecord, ClusterOperationContext context)
+        {
+            if (CanAssertLicenseLimits(context, minBuildVersion: MinBuildVersion54200, serverStore) == false)
+                return;
+
+            if (serverStore.LicenseManager.LicenseStatus.HasRavenEtl)
+                return;
```

Comment:
can this cause inconsistency on different nodes? if some nodes has old license, but others new one?
maybe better to read the license from the storage?

Reply:
Done